src/argparse.cpp

```diff
@@ -82,6 +83,8 @@ void parseArguments(int argc, char **argv, std::vector<std::string>& slot_list,
                       << " DEBUG"
 #endif
                       << "\n";
+            if (!commit_hash.empty())
+               std::cout << "From source hash: " << commit_hash << "\n";
```

Comment:
The thing is, right now, we do not have any "official" release process, right ? We could do it in several ways (add a compilation flag is the simplest). Or do we include the commit hash only to debug releases ?